jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -168,6 +168,8 @@ def __init__(self, *args, **kwargs):
         self.hub.subscribe(self, GlobalDisplayUnitChanged,
                            handler=self._on_global_display_unit_changed)
 
+        # TODO: delete once parallel framework is completed
```

Comment:
In case we don't get to this soon, let's just leave a reference here from the follow-up ticket

```suggestion
```